app/gui/src/main/scala/org/bitcoins/gui/dlc/dialog/CreateDLCOfferDialog.scala

```diff
@@ -369,7 +369,7 @@ class CreateDLCOfferDialog extends Logging {
           decompOpt = Some(digitDecomp)
 
           val addPointButton: Button = new Button("+") {
-            onAction = _ => addPointRow()
+            onAction = _ => addPointRow(isEndPoint = false)
```

Comment:
These can be endpoints

Endpoint doesn't refer to at then end of the list but rather, if it is an endpoint in the payout curve

Reply:
Can a payout curve have more than 2 endpoints, or is 2 the limit?

If I wanted to do a payout for values [10, 100] with only 100 paying me out, would I set up the points like:

<img width="406" alt="Screen Shot 2021-07-06 at 2 08 45 PM" src="https://user-images.githubusercontent.com/22351459/124660702-ca2ce300-de63-11eb-9981-d763832886a3.png">
?

Reply:
Ok, seeing more of what 'Endpoint' impacts. I'll remove the value change.
<img width="1021" alt="Screen Shot 2021-07-06 at 3 10 58 PM" src="https://user-images.githubusercontent.com/22351459/124667724-06b10c80-de6d-11eb-9de8-a847f11466fd.png">